# Cartesian waypoints for end-effector
float64[] start_conf
float64[] goal_conf
geometry_msgs/PoseArray waypoints
---
# Result
bool plan_success
bool execution_success
trajectory_msgs/JointTrajectory planned_trajectory
---
